services/events.service.js

```diff
@@ -339,6 +322,71 @@ class EventsService extends BaseService {
 
     return response
   }
+
+  async doListFromFirebase(eventParams) {
+    let allEvents = []
```

Comment:
cuando hacemos push podemos usar const
```suggestion
    const allEvents = []
```